starlite/cli/_utils.py

```diff
@@ -5,9 +5,10 @@
 import sys
 from dataclasses import dataclass
 from functools import wraps
+from itertools import chain
 from os import getenv
 from pathlib import Path
-from typing import TYPE_CHECKING, Any, Callable, Iterable, Sequence, TypeVar, cast
+from typing import TYPE_CHECKING, Any, Callable, Iterable, Sequence, TypeVar, cast, Generator
```

Comment:
```suggestion
from typing import TYPE_CHECKING, Any, Callable, Generator, Iterable, Sequence, TypeVar, cast
```

Reply:
Shouldn't this have been autofixed?

Reply:
I'd have thought so..

Reply:
think `cli` needs to be added to `tool.ruff.src` in pyproject

Reply:
Hm, no, it seems to run on that file in general.

Reply:
Well.. Turns out we didn't enable the isort rules for ruff.

Reply:
ok well that was the only dart I had to throw :)